app/scripts/platforms/extension.js

```diff
@@ -160,10 +162,11 @@ export default class ExtensionPlatform {
 
     const url = getBlockExplorerLink(txMeta, rpcPrefs);
     const nonce = parseInt(txMeta.txParams.nonce, 16);
+    const view = getURLHostName(url).replace(/([.]\w+)$/u, '');
```

Comment:
- Is this way of deriving view name from URL good enough ?
- `startCase(toLower(view))` this conversion can also be probably done here.